frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/details-card.tsx

```diff
@@ -53,6 +63,21 @@ const DetailsCard: React.FC<DashboardItemProps> = ({
     InfrastructureModel,
     'cluster',
   );
+  const [ocsSubscription, ocsSubscriptionLoaded, ocsSubscriptionError] = useK8sWatchResource<
+    SubscriptionKind
+  >(SubscriptionResource);
+  const [packageManifest, packageManifestLoaded, packageManifestError] = useK8sGet<
+    PackageManifestKind
+  >(PackageManifestModel, 'ocs-operator', 'openshift-storage');
+  const currentChannel =
```

Comment:
```suggestion
 const memoizedSubscription = useDeepCompareMemoize(ocsSubscription, true);
  const currentChannel =
```
Should we memoize these watched resources as they can actually trigger rerenders?

Reply:
But as i see, without memoize too, rerenders are happening. @bipuladh

Reply:
With memoize the renders do not happen, we don't want unnessary re-renders.